obs-tools/opensuse42_submit.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+LIST=`osc list openSUSE:Factory | grep -i -e yast2 -e skelcd -e "^libstorage$" -e perl-bootloader -e linuxrc`
+
+for i in $LIST; do
+  # compare if something is new by comparing binaries ( as version is part of rpm output )
+  OS42=$(osc list -b openSUSE:42 $i -r standard -a x86_64)
+  FACTORY=$(osc list -b openSUSE:Factory $i -r standard -a x86_64)
+  if [ "$OS42" == "$FACTORY" ]; then
+     continue
+  fi
+  echo "submitting $i..."
+  osc sr openSUSE:Factory $i openSUSE:42 -m "opensuse42_submit.sh: update of yast2 related package to the SLE12SP1 version"
```

Comment:
I explained it to lslezak in office, so lets retry it if it also make sense for you. Idea of openSUSE42 is to keep core parts like e.g. installer close to SLE. So if I mention factory it looks like I am trying to use some unstable package. If I use SP1 there, it more invoke we really submit something that we think is stable enough, that we even send to our enterprise linux. So more marketing message :)